src/azul/plugins/metadata/hca/aggregate.py

```diff
@@ -71,7 +71,7 @@ def _transform_entity(self, entity: JSON) -> JSON:
                     is_intermediate=entity['is_intermediate'],
                     count=(fqid, 1),
                     content_description=entity['content_description'],
-                    matrix_cell_count=(fqid, entity['matrix_cell_count']))
+                    matrix_cell_count=(fqid, entity.get('matrix_cell_count')))
```

Comment:
Seems a bit odd to use `.get` here when the other entity values are obtained by key. But it makes sense to use it when wanting to rely on the `None` default.

Reply:
Odd, however required since this PR makes the `matrix_cell_count` field conditionally added (only if an analysis file) to the `files` inner entity.